shared/firestore-models/util.ts

```diff
@@ -1,24 +1,15 @@
 import * as firebase from 'firebase-admin';
 
-import {getConfig, BranchManagerRepoConfig} from './repo-config'
+import {getConfig, BranchManagerRepoConfig, BranchManagerRepoConfigBranch} from './repo-config'
 import {BranchManagerPullRequest} from './pull-request'
 
-/** Determine when a label is one of the target labels based on the repos config. */
-export async function isConfiguredTargetLabel(configId: string, label: string): Promise<boolean> {
-  const configDoc = await getConfig(configId);
-  if (configDoc.exists) {
-    const config = configDoc.data() as BranchManagerRepoConfig;
-    return !!config.branches.find(branch => branch.label === label);
-  }
-  return false;
-}
-
 /** Retrieve the target branch for a pull request based on the repos config. */
-export async function getTargetBranch(
-    configId: string, pullRequestRef: firebase.firestore.DocumentReference): Promise<string> {
+export async function getPullRequestTargetForConfig(
```

Comment:
Does it make more sense to rename `BranchManagerRepoConfigBranch` to `BranchManagerRepoConfigTarget` and then this function would become `getTargetForPullRequest`

Thoughts?

Reply:
That sounds good as well. 

Somehow it still feels not explicit because `Target` could be everything. Bazel target? 😄, but since we require a bit context for the source-code, I think `getTargetForPullRequest` is also fine.